src/components/BuildDetailPage/BuildDetailPage.tsx

```diff
@@ -201,8 +203,8 @@ export const BuildDetailPage = () => {
               title={buildEntityAttributes.scmBuildConfigRevision.title}
               tooltip={buildEntityAttributes.scmBuildConfigRevision.tooltip}
             >
-              {(serviceContainerBuild.data as any)?.scmBuildConfigRevision && (
-                <CopyToClipboard isInline>{(serviceContainerBuild.data as any).scmBuildConfigRevision}</CopyToClipboard>
+              {(serviceContainerBuild.data as Build)?.scmBuildConfigRevision && (
```

Comment:
`serviceContainerBuild.data?.scmBuildConfigRevision`

The purpose of the `any` check was that `serviceContainerBuild.data` has automatically inferred its type (in this case, `Build`). But at the time that I created the PR, `scmBuildConfigRevision` property was not available in the `Build` entity type because the `pnc-api-types` dependency was out-dated.  But now that it's updated, no type cast is needed, because `scmBuildConfigRevision` is known to be on the Build entity for the compiler. And `serviceContainerBuild.data` is detected as `Build` so that typecast is useless.

Reply:
Got it, thanks, fixed